packages/backend/unique-ids-generator/__tests__/unique-ids-generator.test.ts

```diff
@@ -0,0 +1,30 @@
+import { generateUniqueId } from "../src/generate-unique-id";
+
+describe("generate-unique-id", () => {
+  test("Can generate a unique id of default length", () => {
+    const uniqueId = generateUniqueId({});
+
+    expect(uniqueId.length).toBe(4);
+  });
+
+  test("Can generate a unique id of length 6", () => {
+    const uniqueId = generateUniqueId({ length: 6 });
+
+    expect(uniqueId.length).toBe(6);
+  });
+
+  test("Can generate a unique id using the default dictionary", () => {
+    const uniqueId = generateUniqueId({});
+
+    // Default dictionary only includes letters and digits 0-9
+    expect(/^[A-Za-z0-9]*$/.test(uniqueId)).toBeTruthy();
+  });
+
+  test("Can generate a unique id not using the default dictionary", () => {
+    // Dictionary only including letters
+    const LETTERS_ONLY = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+    const uniqueId = generateUniqueId({ dictionary: LETTERS_ONLY });
+
+    expect(/^[A-Za-z]*$/.test(uniqueId)).toBeTruthy();
```

Comment:
Can we also check that it's falsy for numbers?

Reply:
If it contains numbers `.test()` is going to return false!